src/interactiveList.js

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line import/prefer-default-export
+export const checkoffItem = (todoListItem) => {
+  const todoListLi = todoListItem;
+  const checkbox = todoListItem.querySelector('.checkbox');
+  const input = todoListItem.querySelector('.item-description-input');
+
+  const todoContainer = todoListItem.closest('.todo-list');
+
+  if (checkbox.checked) {
+    todoListLi.dataset.completed = true;
+    input.classList.add('checked');
+  } else {
+    todoListLi.dataset.completed = false;
+    input.classList.remove('checked');
+  }
+
+  // Indexes to remove
+  const indexesToRemove = [];
+  todoContainer.querySelectorAll('.todo').forEach((todoNode) => {
+    if (todoNode.dataset.completed === 'true') {
+      indexesToRemove.push(+todoNode.dataset.index);
+    }
+  });
+
+  return indexesToRemove;
```

Comment:
- Hello @rozhi-98, just as the previous reviewers have suggested, please make sure that the `completed` status updates in the local storage when the checkbox is clicked, please fix this. Also, consider ensuring that on page reload if there are tasks in the local storage it populates the UI 👍 👍.

Your project:
<img width="957" alt="erfrvr" src="https://user-images.githubusercontent.com/73607512/206877611-87385ca6-c8aa-45c9-8dcb-9b4bb8f085ef.PNG">


**OPTIONAL** _ please remove all disabled linter checks, remember linters help us ensure that we write code that is clean and easy to maintain, please fix this 👍 👍